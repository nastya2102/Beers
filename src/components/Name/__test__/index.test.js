import React from 'react'
import { render } from '@testing-library/react'
import Name from '../index'

describe('Name', () => {
  test('should render Name component', () => {
    const { container } = render(<Name onClick={() => {}}><span>123</span></Name>)

    expect(container).toBeTruthy()
  })

  test('should match snapshot Name component', () => {
    const { container } = render(<Name onClick={() => {}}><span>123</span></Name>)

    expect(container).toMatchSnapshot()
  })
})
