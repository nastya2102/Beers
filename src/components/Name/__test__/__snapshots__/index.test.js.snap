// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name should match snapshot Name component 1`] = `
<div>
  <div
    class="sc-bdfBwQ tXWSF"
  >
    <span>
      123
    </span>
  </div>
</div>
`;
